content/patterns/multicloud-gitops-sgx-hello-world/mcg-sgx-hello-world-getting-started.adoc

```diff
@@ -0,0 +1,325 @@
+----
+`<DOCKER_REPOSITORY>` is name a docker repository, `<TAG>` is a version of an image
+
+.. Login to Docker Hub:
++
+[source,terminal]
+----
+$ docker login -u=<DOCKER_ID> docker.io 
+----
+`<DOCKER_ID>` is your login to Docker Hub
+
+.. Push the image to registry:  
++
+[source,terminal]
+----
+$ docker push <DOCKER_REPOSITORY>:<TAG>
+----
+. Update the file _charts/all/hello-world-sgx/values.yaml_ with the information about your image in your forked repo with pattern.
+
+[id="deploying-cluster-using-patternsh-file"]
+== Deploying the cluster by using the pattern.sh file
+
+To deploy the cluster by using the _pattern.sh_ file, complete the following steps:
+
+. Login to your cluster by running the following command:
++
+[source,terminal]
+----
+oc login
+----
++
+Optional: Set the `KUBECONFIG` variable for the _kubeconfig_ file path:
++
+[source,terminal]
+----
+export KUBECONFIG=~/<path_to_kubeconfig>
+----
+
+. Deploy the pattern to your cluster. Run the following command:
++
+[source,terminal]
+----
+./pattern.sh make install
+----
+
+Verify that the Operators have been installed.
+
+. To verify, in the *OpenShift Container Platform* web console, navigate to *Operators → Installed Operators* page.
+. Check that the following Operators are installed with  `Succeeded` status (Figure 1): 
+* *Advanced Cluster Management for Kubernetes* 
+* *Intel Device Plugins Operator* 
+* *multicluster engine for Kubernetes* 
+* *Node Feature Discovery Operator* 
+* *Red Hat Openshift GitOps*
+* *Validated Patterns Operator*
++
+.List of Installed Operators for Multicloud GitOps Validated Pattern with SGX
+image::multicloud-gitops-sgx-hello-world/sgx-installed-operators.png[Multicloud GitOps Hub with SGX installed operators,role="related thumb right"]
++
+
+
+== Deploying the cluster by using the Validated Patterns Operator
+
+To install the *Validated Patterns Operator*:
+
+. Log in to the *Openshift Container Platform* web console and select *Operators > OperatorHub*.
+
+. Search for *Validated Patterns Operator*, open it and click *Install*.
++
+.Install Validated Patterns Operator
+image::multicloud-gitops-sgx-hello-world/sgx-validated-pattern-operator.png[Install Validated Patterns Operator,scale=50]
++
+. Choose default settings for the installation mode, namespaces and update strategy and confirm it by clicking *Install*.
+
+. Select *Operators > Installed Operators*.
+
+. Ensure that *Validated Patterns Operator* is listed in the `openshift-operators` project with a status `Succeeded`.
+
+After succeeded installation open *Validated Patterns Operator*, go to *Pattern* tab and click *Create Pattern*.
+
+. Fill the *Name* of the pattern `multicloud-gitops-sgx-hello-world` and *Cluster Group Name* `hub` (from _values-global.yaml_ file).
+
+. Under *Git Config > Target Repo* copy the link to your fork and under *Git Config > Target Revision* write the name of your branch (Figure 3).
+
+. Click *Create* to create the pattern.
```

Comment:
It is a bit hard to understand. I get it, but only thanks to the image.
Can we rephrase it? Maybe sth like:

1. Set the _Name_ field to `multicloud-gitops-amx-rhoai`, and the _Cluster Group Name_ to `hub`. Values must be the same as in the _values-global.yaml_ file (Figure 3).
2. As a *Git Config > Target Repo* value, paste the link to your fork. Under *Git Config > Target Revision* write the name of your branch (Figure 3).
3. Click the _Create_ button to create the pattern.

Reply:
Updated